x-pack/plugins/enterprise_search/README.md

```diff
@@ -22,7 +22,8 @@ Unify all your team's content into a personalized search experience. https://www
 
 1. When developing locally, Enterprise Search should be running locally alongside Kibana on `localhost:3002`.
 2. Update `config/kibana.dev.yml` with `enterpriseSearch.host: 'http://localhost:3002'`
-3. For faster QA/development, run Enterprise Search on [elasticsearch-native auth](https://www.elastic.co/guide/en/app-search/current/security-and-users.html#app-search-self-managed-security-and-user-management-elasticsearch-native-realm) and log in as the `elastic` superuser on Kibana.
+
+Problems? If you're an Elastic Enterprise Search engineer, please reach out to @elastic/enterprise-search-frontend for questions or our in-depth Getting Started developer guide.
```

Comment:
I have mixed feelings about this line. It's definitely helpful for Elastic engineers, but unwelcoming for the community. I don't know how we can achieve both. Maybe drop it altogether? Elastic engineers should have an understanding of internal teams/channels.

Reply:
Do we expect community contributions to our plugin? How would they run our dev Kibana plugin without dev ent-search? 🤷 I guess you could contribute UI fixes, but not API? TBH our frontend generally isn't well set up for open source contributions due to our internal APIs.

Reply:
> Elastic engineers should have an understanding of internal teams/channels.

@vadimkibana this was in an App Search retro, but we had @kovyrin get extremely frustrated while working in Kibana because he didn't see the Getting Started guide pinned to the Slack channel. I don't think we should assume engineers know what Slack channel we're in or only mention the guide in Slack only, but ensure people know how to reach out to us if necessary.

Reply:
@constancecchen I think you meant @yakhinvadim

Reply:
🤦‍♀️ Argghh super sorry for the incorrect ping!

Reply:
Ok, let's keep the line.

Reply:
I think this is a super interesting q on how we want to consider/handle open source contributions in the future though - will bring this up in the larger eng channel if that's cool!